pkg/cassandra/nodetool/nodetool.go

```diff
@@ -135,17 +124,6 @@ func (t *tool) Status() (NodeMap, error) {
 		)
 	}
 
-	if !mappedNodes.IsSuperset(leavingNodes.Union(joiningNodes).Union(movingNodes)) {
-		return nil, fmt.Errorf(
-			"mapped nodes must be a superset of leaving, joining and moving nodes. "+
-				"Leaving: %v, "+
-				"Joining: %v, "+
-				"Moving: %v, "+
-				"Mapped: %v",
-			leavingNodes, joiningNodes, movingNodes, mappedNodes,
-		)
-	}
-
```

Comment:
This function confuses me a lot, and I'm not really sure how it works.

Can you add a comment to the function itself explaining the rough algorithm in place here, so that this change can be reviewed properly?

Reply:
I've documented the function. Please take another look.